Source/Genetic.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, mutRate, mutType, popSize, crossType, cutoff, gridSize):
         self.crossType = crossType
         self.cutoff = cutoff
         self.gridSize = gridSize
-        self.NumberofNumbers = 3
+        self.NumberofNumbers = 15
```

Comment:
I see we are passing on the tradition of waiting for someone else to un-hardcode this

Reply:
Oh i didn't mean to add this file, whoops